project/Dependencies.scala

```diff
@@ -183,6 +185,12 @@ object Dependencies {
   )
 
   val azureDependencies: List[ModuleID] = List(
+    "com.azure" % "azure-storage-blob" % azureStorageBlobV
```

Comment:
I think this may not be necessary anymore? We are only using the NIO library at this point